README.md

```diff
@@ -12,220 +13,242 @@ This is Kubernetes Cheatsheet based on Kubernetes API 1.19 version.
+| Create deployment yaml file           | kubectl create deployment `<deployment_name>` --image `<image_name>` --dry-run=client -o yaml > `<file_name>`.yaml                                             |
+| Create service                        | kubectl create service `<service-type>` `<service_name>` --tcp=`<port:target_port>`                                                                            |
+| Create service yaml file              | kubectl create service `<service-type>` `<service_name>` --tcp=`<port:target_port>` --dry-run=client -o yaml > `<file_name>`.yaml                              |
+| Expose service from pod/deployment    | kubectl expose deployment `<pod/deployment_name>` --type=`<service-type>` --port `<port>` --target-port `<target_port>`                                        |
+| Create config map from key-value      | kubectl create configmap `<configmap_name>` --from-literal=`<key>:<value>` --from-literal=`<key>:<value>`                                                      |
+| Create config map from file           | kubectl create configmap `<configmap_name>` --from-file=`<file_name>`                                                                                          |
+| Create config map from env file       | kubectl create configmap `<configmap_name>` --from-env-file=`<file_name>`                                                                                      |
+| Create secret from key-value          | kubectl create secret generic `<secret_name>` --from-literal=`<key>:<value>` --from-literal=`<key>:<value>`                                                    |
+| Create secret from file               | kubectl create secret generic `<secret_name>` --from-file=`<file_name>`                                                                                        |
+| Create job                            | kubectl create job `<job_name>` --image=`<image_name>`                                                                                                         |
+| Create job from cronjob               | kubectl create job `<job_name>` --from=`cronjob/<cronjob-name>`                                                                                                |
+| Create cronjob                        | kubectl create cronjob --image=`<image_name>` --schedule='`<cron-syntax>`' -- `<command>` `<args>`                                                             |
+| Create inline yaml                    | cat <<EOF \| kubectl create -f - `<enter>` <br> YAML CONTENT `<enter>` <br> EOF `<enter>`                                                                      |
+| Create role                           | kubectl create role `<role_name>` --verb=`<verb>` --resource=`<resource.group/subresource>` --resource-name=`<resource_name>` --dry-run=`<server/client/none>` |
```

Comment:
I think we have "create role" command already. Could you go to the RBAC section?

Reply:
I change it to autoscaling command